examples/densenet_pytorch/README.md

```diff
@@ -0,0 +1,52 @@
+# PyToch serving  
+This example shows how to serve PyTorch trained models for flower species recognition..
+The custom handler is implemented in `densenet_service.py`.
+For simplicity, we'll use a pre-trained model. For simplicity we will use docker container to run Model Server.
+
+## Getting Started With Docker
+Build the docker image with pytorch as backend engine:
+```bash
+  cd examples/densenet_pytorch/
+  docker build . -t mms_with_pytorch
+```
+
+Run the container that you have built in previous step.
+```bash
+  docker run -it --entrypoint bash mms_with_pytorch
```

Comment:
Docker commands are not required, you seem to be running "mxnet-model-server" as is below.